src/Strategy/DateTimeFormatterStrategy.php

```diff
@@ -86,18 +101,27 @@ public function extract($value)
      * @param mixed|string $value
```

Comment:
It has no sense to have `mixed|string` as string is also mixed. If we want to specify all types we should have here also probably `DateTimeInterface` and `null` or just leave `mixed`.

Reply:
It does make sense, though. If a DateTimeInterface is provided as the $value, a string is returned. Otherwise, mixed values are returned verbatim. I'll update the return annotation to reflect this.